src/core/thread/router_table.hpp

```diff
@@ -312,20 +312,21 @@ class RouterTable : public InstanceLocator
     bool IsAllocated(uint8_t aRouterId) const;
 
     /**
-     * This method updates the router table with a received Route TLV.
+     * This method updates the Router ID allocation.
      *
-     * @param[in]  aTlv  A reference to the Route TLV.
+     * @param[in]  aRouterIdSequence  The Router Id Sequence.
+     * @param[in]  aRouterIdSet       A reference to the Router Id Set.
      *
      */
-    void ProcessTlv(const Mle::RouteTlv &aTlv);
+    void ProcessRouterIdSet(uint8_t aRouterIdSequence, const Mle::RouterIdSet &aRouterIdSet);
```

Comment:
How about renaming this to `UpdateRouterIdSet()`?
(it is fine for me as is, just a suggestion).

Reply:
renamed as your suggestion.